doc/transfer-format.rst

```diff
@@ -0,0 +1,109 @@
+Key Transfer Format
+===================
+
+This document describes a format recommendation for secret key backup
+attchments. The Autocrypt Key Transfer Format is designed to work as an
+attachment in a mail client that can be handled automatically by Autocrypt
+clients, but also strives for usability when saved as a file in agnostic
+clients.
+
+This goal is achieved by two mechanisms:
+
+- The MIME part MUST have a Content-Disposition of "attachment". Its filename,
+  as specified in Content-Type and Content-Disposition header parameters, MUST
+  have an extension of ".html".
+- The file MUST be formatted as proper HTML. It SHOULD give a human-readable
+  description at the beginning, and MUST include the encrypted key as a
+  contiguous block of ASCII-armored data, with no indentation or HTML tags
+  inside. This block SHOULD reside inside a <pre> block for readability.
+
+With this technique, the way the attachment is handled when saved to disk by the
+user is independent from its handling by clients aware of its Content-Type. HTML
+as the actual file content is chosen as a universal and familiar format. It can
+be opened in a browser, allowing the file to provide a description of itself "on
+double-click" on almost any system.
+
+Since the file contains an easy to locate block of ASCII-armored data, it can
+still easily be processed by OpenPGP tools. In particular, GnuPG ignores the
+header before the armored data starts, so a regular call to :code:`gpg
+--decrypt` still works.
+
+An exemplary secret key backup MIME part would look as such:
+
+.. code-block:: html
+
+  Content-Type: application/pgp-key-backup; name="Autocrypt-Key for name-at-example.org.html"
+  Content-Description: OpenPGP Digital Signature
+  Content-Disposition: attachment; filename="Autocrypt-Key for name-at-example.org.html"
+
+  <html><body><h1>Autocrypt Key</h1><p>
```

Comment:
Instead of talking about "key" here and below i suggest to rather talk about "Autocrypt secret key number" or so.  Basically a term that makes some sense for someone who has never heart of "keys" and who may even use some autocrypt-by-default client (so even less context to be assumed).